src/commands/key-value/del.ts

```diff
@@ -27,7 +27,7 @@ export default class DeleteKey extends NimBaseCommand {
 
     static args = [{ name: 'key', description: 'The key to be deleted', required: true }]
 
-    static aliases = ['kv:del', 'kv:delete']
+    static aliases = ['key-value:delete']
```

Comment:
do we lose kv:del and kv:delete in favor of only key-value:delete?

Reply:
no, in fact, `kv:del` was unnecessary, as `del` is the name of the command itself, doesn't need an alias.
`kv` is a topic alias, it will continue working, these all are valid
- kv del mykey
- kv delete mykey
- key-value del mykey
- key-value delete mykey

Reply:
thanks for the explanation 👍